src/frontend/components/_features/[workspace]/global-variable-list/index.tsx

```
@@ -0,0 +1,94 @@
+import { useCallback, useEffect, useState } from 'react'
+
+import { useOpenPLCStore } from '../../../../store'
+import { serializeGlobalVariableListToText } from '../../../../utils/PLC/global-variable-list-serializer'
+import { parseGlobalVariableListFromText } from '../../../../utils/PLC/global-variable-list-text-parser'
+import { toast } from '../../[app]/toast/use-toast'
+import { VariablesCodeEditor } from '../../../_organisms/variables-code-editor'
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)